src/tests/ftest/io/mdtest_small.yaml

```diff
@@ -17,14 +17,18 @@ pool:
     createset:
         group: daos_server
     createsize:
-        scm_size: 300000000
+        scm_size: 1600000000
```

Comment:
this pool size seems pretty large..

Reply:
That itself seems odd, I'd expect IOR to use more space than mdtest (although it depends on params obviously) and the same test over dfs/dfuse should use almost exactly the same.

Reply:
Reducing it will make the test fail. If this is a concern I can open a separate ticket to investigate this.

Reply:
i saw it first as 16g which i thought it was large. i see it's 1.6 which is fine.
but depending on the number of files and file size, you might be legitimately running out of space.

Reply:
Regardless of the absolute size it's still five times bigger than the previous value, does this imply that mdtest over dfuse takes five times as much space?  As I understand it this is really about enabling essentially the same test over a different transport layer, the space used should remain the same so this is something to look into.

Yes please to a seperate ticket to investigate this.  Does either the mdtest-over-dvs or the mdtest-over-dfuse test log the amount of space used?